sympy/functions/special/delta_functions.py

```diff
@@ -31,6 +49,38 @@ class DiracDelta(Function):
 
     5) ``DiracDelta(x,k) = 0``, for all ``x != 0``
 
+    Examples
+    ========
+
+    >>> from sympy import DiracDelta, diff, pi, Piecewise
+    >>> from sympy.abc import x, y
+
+    >>> DiracDelta(x)
+    DiracDelta(x)
+    >>> DiracDelta(1)
+    0
+    >>> DiracDelta(-1)
+    0
+    >>> DiracDelta(pi)
+    0
```

Comment:
This is clear from the docstring I guess, anyways its not a big issue. I just don't like cluttering the docstrings with almost similar examples, that is ofcourse my own opinion.